outputs.tf

```diff
@@ -0,0 +1,4 @@
+resource "okta_group_memberships" "has_nice_things" {
```

Comment:
What is this one for? It shouldn't be producing any outputs. Did you mean to just put this in `main.tf`?